packages/core/demo/data/line.ts

```diff
@@ -77,6 +79,34 @@ export const lineTimeSeriesOptions = {
 	curve: "curveMonotoneX"
 };
 
+export const lineTimeSeriesWithThresholdsOptions = {
+	title: "Line with thresholds (time series)",
+	axes: {
+		bottom: {
+			title: "2019 Annual Sales Figures",
+			mapsTo: "date",
+			scaleType: "time",
+			thresholds: [
```

Comment:
can this potentially cause issues if you define thresholds on both bottom & top axes?

Reply:
Do you mean, if we have both the bottom and top axes defined and with thresholds?
Something like:
```javascript
export const lineTimeSeriesWithThresholdsOptions = {
	title: "Line with thresholds (time series)",
	axes: {
		bottom: {
			mapsTo: "date",
			scaleType: "time",
			thresholds: [ {value: new Date(2019, 0, 11), label: "Bottom"}]
		},
		top: {
			scaleType: "time",
			thresholds: [
				{value: new Date(2019, 0, 12), label: "Top", fillColor: "white"}
			]
		},
		right: {
			thresholds: [
				{value: 10000, fillColor: "green", label: "Right"}
			]
		},
		left: {
			mapsTo: "value",
			scaleType: "linear",
			thresholds: [
				{value: 55000, label: "Left", fillColor: "orange"},
				{value: 10000, fillColor: "yellow", label: "Left"}
			]
		}
	},
	curve: "curveMonotoneX"
};
```
<img width="403" alt="Screenshot 2020-04-28 at 12 41 38" src="https://user-images.githubusercontent.com/5892387/80478347-ae6baa00-894d-11ea-8046-2beb521a4f5d.png">

The only problem I see is that if you define the same threshold value for the same orientation (right, left in the example above), the first threshold will override the second one defined. 
But it is left to the user's common sense, what do you think?
Do you see any other issues? 

Did I address your question or did you mean something different @theiliad? 



I've found an issue (as you can see from the screenshot) on horizontal thresholds scale value when both axes are present. I'm working on it!

Reply:
(Fixed! 🙂)
<img width="308" alt="Screenshot 2020-04-28 at 18 02 02" src="https://user-images.githubusercontent.com/5892387/80509969-6151fd00-897a-11ea-8336-312c58a5c4f3.png">

Reply:
🙂